dipy/core/gradients.py

```diff
@@ -244,3 +244,55 @@ def gradient_table(bvals, bvecs=None, big_delta=None, small_delta=None,
                                            small_delta=small_delta,
                                            b0_threshold=b0_threshold,
                                            atol=atol)
+
+
+def reorient_bvecs(gtab, rots):
+    """
+    Reorient the directions in a GradientTable according
+
+    When correcting for motion, rotation of the diffusion-weighted volumes
+    might cause systematic bias in rotationally invariant measures, such as FA
+    and MD, and also cause characteristic biases in tractography, unless the
+    gradient directions are appropriatel reoriented to compensate for this
+    effect [Leemans2009]_.
+
+    Parameters
+    ----------
+    gtab : a GradientTable class instance.
+        The nominal gradient table with which the data were acquired.
+    rots : (n, 3) array.
```

Comment:
I am worried that some people may use this as a black box (I'm not sure if we should add a warning here or be more specific about what the `rots` are) because there may be an issue regarding how the rotational parameters are given by different software. For example, flirt (and mcflirt, and I assume all FSL software, like eddy) returns a matrix M that maps points in physical space from the moving image towards the static image  (that is the most intuitive way for most people, because it is the moving image which is being "sent" to the static image). ANTS, on the other hand, returns a matrix that maps points in the static image towards points in the moving image, but this choice makes sense too: when we transform the moving image towards the static image, we actually send each point `x` in the static image towards its corresponding `y` in the moving image (contrary to the common intuition) so we can interpolate the moving image at `y` to obtain the intensity value that must be written at each `x` to form the transformed image.  So, whenever Flirt needs to transform an image, it actually computes the inverse of the matrix it returned, while ants simply uses the returned matrix itself. However, if people used Flirt, they must apply the returned matrix unchanged to rotate the bvecs, but if for some reason they used ANTS they need to apply the inverse of the returned matrix. Another issue is whether the transformation was computed in RAS or LPS (or any other) coordinate system, and if the voxels in the image are given in C order of Fortran order. The dangerous thing is that the transforms are often very close to the identity, which makes it very hard to notice when we are making a mistake.